CONTRIBUTING.md

```diff
@@ -61,7 +61,7 @@ We actively welcome your pull requests.
 If you plan to contribute a change based on an open issue, please assign your self by commenting on the following word `.take`. Issues that are not assigned are assumed open, and to avoid conflicts, please assign yourself before beginning work on any issues.  
```

Comment:
I think also the word yourself should be joined

Reply:
sorry, @filiptronicek I missed this one.

Reply:
The important thing is to catch all of the typos ;), happy to help a bit too